util/ctassert.h

```diff
@@ -14,6 +14,7 @@
 /* Define using libcperciva namespace to avoid collisions. */
 #define CTASSERT(x)			libcperciva_CTASSERT(x, __LINE__)
 #define libcperciva_CTASSERT(x, y)	libcperciva__CTASSERT(x, y)
-#define libcperciva__CTASSERT(x, y)	extern char libcperciva__assert ## y[(x) ? 1 : -1]
+#define libcperciva__CTASSERT(x, y)	extern char	\
+    libcperciva__assert ## y[(x) ? 1 : -1]
```

Comment:
Better to break this earlier, i.e.
```
#define libcperciva__CTASSERT(x, y) \
    extern char libcperciva__assert ## y[(x) ? 1 : -1]
```

Same goes for parsenum below -- the "most natural" place to line wrap is where we have tabs.